docs/UserGuide.adoc

```diff
@@ -49,27 +49,27 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 *Command Legend*
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `open NAME`, `NAME` is a parameter which can be used as `open Notably`.
-* Items in square brackets are optional e.g `-t TITLE [-b BODY]` can be used as `-t Notably -b Lorem ipsum` or as `-t Notably`.
-* Parameters can be in any order e.g. if the command specifies `-t TITLE -b BODY`, `-b BODY -t TITLE` is also acceptable.
+* Items in square brackets are optional e.g both `open -t Notably` and `open Notably` will open the note titled "Notably".
```

Comment:
Hmmm I feel like we need to show the user what is the notation (what we mean by square brackets). For this case, we should show the user that the command is `open [-t] TITLE`

Reply:
Will rephrase!